samples/ResourceOwnerPasswordFlow/Certificate.cer

```diff
@@ -0,0 +1,24 @@
+Bag Attributes
```

Comment:
Hmm. Supporting those targets is a nice-to-have. So, I would like to keep them for the time that supporting them becomes a priority.